pass-nihms-loader/nihms-data-transform-load/src/test/java/org/eclipse/pass/loader/nihms/entrez/EntrezPmidLookupTest.java

```diff
@@ -15,44 +15,81 @@
  */
 package org.eclipse.pass.loader.nihms.entrez;
 
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
+import org.apache.commons.io.IOUtils;
+import org.eclipse.pass.loader.nihms.NihmsTransformLoadCLIRunner;
 import org.json.JSONObject;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.TestPropertySource;
 
 /**
  * @author Karen Hanson
- * @version $Id$
  */
+@SpringBootTest
+@TestPropertySource("classpath:test-application.properties")
+@WireMockTest(httpPort = 9911)
 public class EntrezPmidLookupTest {
 
+    @Autowired
+    protected PmidLookup pmidLookup;
+
+    // Needed so tests can run after application starts
+    @MockBean private NihmsTransformLoadCLIRunner nihmsTransformLoadCLIRunner;
+
+    @Value("${pmc.entrez.service.url}")
+    private String ENTREZ_PATH;
```

Comment:
I think this can be deleted, not being used looks like.

Reply:
Done.